.vscode/launch.json

```diff
@@ -97,8 +102,15 @@
       ],
       "stopOnEntry": false,
       "sourceMaps": true,
+      "smartStep": true,
       "outFiles": ["${workspaceFolder}/packages/*/out/**/*.js"],
-      "preLaunchTask": "Compile"
+      "sourceMapPathOverrides": {
+        "../../../../test/vscode-integration": "${workspaceFolder}/packages/salesforcedx-vscode-core/out/test/vscode-integration/*",
+        "webpack:///salesforcedx-utils-vscode/./*": "${workspaceFolder}/packages/salesforcedx-utils-vscode/*",
+        "webpack:///*": "*"
+      },
+      "preLaunchTask": "Compile",
+      "internalConsoleOptions": "openOnSessionStart"
```

Comment:
Adding `intergnalConsoleOptions` to the missing configs allows VSCode to open the terminal and focus in the correct panel (Debug).

Reply:
Is this related to the appveyor test failures we were seeing earlier where we couldn't locate the console output?

Reply:
No, the test failures were caused by the tests executing before the extensions were active. I've fixed that by changing the way we run those tests. This setting will focus in the Debug panel of the terminal once you run the launch config.